### DISCUSSION TITLE
i see in code search page 555k result but only 5 page

### DISCUSSION THREAD

**Discussion by lhandiyouness1133** [community/community#55202] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

hello
please i see in page result of code page More 500k result

but only 5 pages

**Comment by elblasy33** [comment]
I noticed that too

**Comment by look** [comment, marked as answer]
Code search is currently limited to the first 100 results. We will provide an API for exhaustive retrieval.

**Reply by SlackSilencer** [reply]
@look May I ask what is the timeline to remove such limitation; And what is the major reason of having such limitation in new UI.
Thank you

**Reply by zhaojc** [reply]
hope the old search function back

**Comment by sgf** [comment]
I really can't understand Microsoft's behavior. You don't reduce costs through technology, but rely on castration of functions and degrading the experience. This is really puzzling.
